### PR TITLE
fmod scribing

### PR DESCRIPTION
**Pull request by AlekseyPanas**

### CODE REVIEW

File: Assets/Prefabs/PlayerCamera/GestureSystem/GestureSystem.cs [hunk 1]
```diff
@@ -60,13 +65,22 @@ public class GestureSystem : AGestureSystem
...
+            }
+
             Vector2 new_mouse_pos = _controls.Game.MousePos.ReadValue<Vector2>();
             Vector2 scaled_new_mouse_pos = new Vector2(new_mouse_pos.x / Screen.width, new_mouse_pos.y / Screen.height);
             Vector2 scaled_former_mouse_pos = mouseTrack.Count > 0 ? new Vector2(mouseTrack[mouseTrack.Count - 1].x / Screen.width, mouseTrack[mouseTrack.Count - 1].y / Screen.height): scaled_new_mouse_pos;
             float diff_mag = (scaled_former_mouse_pos - scaled_new_mouse_pos).magnitude;
             if (diff_mag > DRAG_DIST_TO_ADD || mouseTrack.Count == 0) {
                 mouseTrack.Add(new_mouse_pos);  // Add user mouse point
                 cum_dist += diff_mag;
+                Debug.Log(diff_mag * 7000);
```
Comment (1whatleytay) [inline Assets/Prefabs/PlayerCamera/GestureSystem/GestureSystem.cs:82]: Please no

**Review by 1whatleytay — CHANGES_REQUESTED** [review]
This is not code

**Review by 1whatleytay — APPROVED** [review]

**Review by 1whatleytay — APPROVED** [review]

File: Assets/Scenes/Desert.unity [hunk 2]
```diff
@@ -4144,7 +4144,7 @@ Transform:
   m_GameObject: {fileID: 727257922}
   serializedVersion: 2
   m_LocalRotation: {x: 0.043768927, y: 0.7582166, z: -0.009299526, w: 0.65046555}
-  m_LocalPosition: {x: 0.868, y: 2.564, z: 0.672}
+  m_LocalPosition: {x: 0.868, y: 1.9999999, z: 0.672}
   m_LocalScale: {x: 1, y: 1, z: 1}
   m_ConstrainProportionsScale: 0
   m_Children: []
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:4150]: children?

File: Assets/Scenes/Desert.unity [hunk 3]
```diff
@@ -5840,16 +5840,31 @@ PrefabInstance:
...
+      objectReference: {fileID: 0}
+    - target: {fileID: 2604257323502352724, guid: 8986b7a9211be254981f42fbaf58582c,
+        type: 3}
+      propertyPath: m_Offset.z
+      value: 0.5390244
+      objectReference: {fileID: 0}
     - target: {fileID: 2604257323502352724, guid: 8986b7a9211be254981f42fbaf58582c,
         type: 3}
       propertyPath: m_FadeScale
-      value: 0.885
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:5846]: red=wrong

File: Assets/Scenes/Desert.unity [hunk 4]
```diff
@@ -10525,20 +10545,67 @@ PrefabInstance:
...
+    Path: event:/Scribing
+  Event: 
+  PlayEvent: 0
+  StopEvent: 0
+  AllowFadeout: 1
+  TriggerOnce: 0
+  Preload: 0
+  AllowNonRigidbodyDoppler: 0
+  Params:
+  - Name: ClickActive
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:10600]: who is clickactive?

File: Assets/Scenes/Desert.unity [hunk 5]
```diff
@@ -5840,16 +5840,31 @@ PrefabInstance:
...
+    - target: {fileID: 2604257323502352724, guid: 8986b7a9211be254981f42fbaf58582c,
+        type: 3}
+      propertyPath: m_Offset.z
+      value: 0.5390244
+      objectReference: {fileID: 0}
     - target: {fileID: 2604257323502352724, guid: 8986b7a9211be254981f42fbaf58582c,
         type: 3}
       propertyPath: m_FadeScale
-      value: 0.885
+      value: 1
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:5856]: 1

File: Assets/Scenes/Desert.unity [hunk 6]
```diff
@@ -5860,6 +5875,11 @@ PrefabInstance:
       propertyPath: m_Layer
       value: 10
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:5876]: 10

File: Assets/Scenes/Desert.unity [hunk 7]
```diff
@@ -10525,20 +10545,67 @@ PrefabInstance:
...
   m_CorrespondingSourceObject: {fileID: 8744320001328329995, guid: 59f0f1001e5bc0149ae4636e27b8dbb7,
     type: 3}
   m_PrefabInstance: {fileID: 1707100391}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 0}
+  m_GameObject: {fileID: 1707100395}
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: 625b0bff2986a40f9b53654ab09b9ab1, type: 3}
   m_Name:
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:10564]: first name: Deez

File: Assets/Scenes/Desert.unity [hunk 8]
```diff
@@ -10525,20 +10545,67 @@ PrefabInstance:
...
+  AllowFadeout: 1
+  TriggerOnce: 0
+  Preload: 0
+  AllowNonRigidbodyDoppler: 0
+  Params:
+  - Name: ClickActive
+    Value: 0
+  - Name: scribe_success
+    Value: 0
+  - Name: scribing_speed
```
Comment (cwityas) [inline Assets/Scenes/Desert.unity:10604]: scribing speed more like scribing deez HA

Comment (cwityas) [reply Assets/Scenes/Desert.unity:10604]: good one

**Review by cwityas — COMMENTED** [review]
nunya

**Review by cwityas — APPROVED** [review]
no bad

**Pull request merged**